master/buildbot/reporters/gerrit_verify_status.py

```diff
@@ -226,6 +229,16 @@ def send(self, build):
                     reporter=reporter,
                     category=category,
                     duration=duration)
+
+                # The verify-status plugin will return 204 NO CONTENT
+                # on success.
+                if response.code != http_client.NO_CONTENT:
+                    try:
+                        body = yield response.text()
```

Comment:
It works for me and does appear to exist according to the docs:

https://treq.readthedocs.io/en/latest/api.html#treq.response._Response.text

Maybe `.text()` was introduced in a later version of treq? Happy to change this to response.content() only if you think it's appropriate.